cypress/integration/navigations.ts

```diff
@@ -0,0 +1,58 @@
+import { IFRAME_SELECTOR } from '../support/constants';
+
+describe('navigations', () => {
+  beforeEach(() => {
+    cy.loadPage('/streamed-epub');
+  });
+
+  it('table of content link should update the page content', () => {
+    cy.iframe(IFRAME_SELECTOR)
+      .findByRole('img', {
+        name: "Alice's Adventures in Wonderland, by Lewis Carroll",
+      })
+      .should('exist');
+
+    // Open TOC menu
+    cy.findByRole('button', { name: 'Table of Contents' }).click();
+
+    // Open chapter 1
+    cy.findByRole('menuitem', { name: 'I: Down the Rab­bit-Hole' }).click();
+    cy.wait(100); // wait for the nested iframe to render
+
+    cy.iframe(IFRAME_SELECTOR)
+      .find('.subtitle')
+      .contains('Down the Rab­bit-Hole');
+  });
+
+  it('next and previous button', () => {
+    // load iframe
+    cy.iframe(IFRAME_SELECTOR);
+
+    cy.findByRole('button', { name: '>' }).click();
```

Comment:
This is actually indicating that we have an A11y bug: we need to give that button a better aria-label: "Next Page" manually. By default buttons use their content as their aria label, but in this case it doesn't make much sense for screen readers, so we should add an explicit one. You can make a ticket for that in the staging sprint and put a `FIXME` comment with the ticket number here